vitrina/smart_contracts/urls.py

```diff
@@ -1,11 +1,25 @@
 from django.urls import path
 
-from vitrina.smart_contracts.views import AgreementCreateView
+from vitrina.smart_contracts.views import (
+    AgreementCreateView,
+    AgreementListView,
+    AgreementDetailView,
+)
 
 urlpatterns = [
     path(
         "projects/<int:pk>/agreement/add/",
         AgreementCreateView.as_view(),
         name="agreement-create",
     ),
+    path(
+        "projects/<int:pk>/agreement/",
```

Comment:
Using full names can make you view more readable. Optional
```suggestion
        "projects/<int:project_id>/agreement/",
```

Reply:
Already existing project URLs in `project` app uses `projects/<int:pk>/...`, so I kept doing the same for consistency: https://github.com/atviriduomenys/katalogas/blob/cbe3c19a20586b2c2626d18cee1e35aed5a8476b/vitrina/projects/urls.py#L24